### ISSUE TITLE
User issue: confirmation is long-winded

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#288]
> This is a very long winded way to contact my MP.
> Is there a quicker route?
> Not sure most would persevere with it.
> Emailed Hilary Benn this am. Direct. E mail address click on write your message and Click....gone.

I do have a set response for this kind of complaint (helps prevent spam; verifies that the response will be sent to the correct address and not a typo, etc) but did I see some discussion somewhere recently (internal blog? IRC?) about ways the verification step might be dropped thanks to new technologies?

Or was that just a beautiful dream?

**Comment by MyfanwyNixon** [comment]
In response to the confirmation email:

> I have already written to them, supplied ALL my details and used the link you provided. Something must have registered otherwise you wouldn't have got my complete letter pasted to your mail. I have no intention to put all my details again into the net, in fact I am somewhat put off to reply again in one of your campaigns, if the letters for some mysterious reason 'get lost'.
Please get your software in order; there are just too many phishermen out there and who knows where my email address, post code and  postal address might end up.

(From a doctor, so presumably someone with a reasonable education!)

**Comment by MyfanwyNixon** [comment]
> I  am diisgusted to receive your email together with a copy of my email that I wrote, in confidence, to my elected Member of Parliameent. What gives you the God given right to intercept/access my emailto my elected member of Parliament and then contact me anonymously under the guise of "writetothem.com". Try "Freedom of Information" next time if you want to know what my elected Member of Parliament is up to. As I see it, this is an invasion of my privacy!!!!!!!
 
Is nothing sacred in the country!!!!

**Comment by MyfanwyNixon** [comment]
> Tried to send details to my M.P. but you need confirmation that I want to send it.  I cannot think of any reason as to why I would go to this trouble if I did not want him to get it.

**Comment by MyfanwyNixon** [comment]
> Do not send, I find this email incomprihensible!

**Comment by MyfanwyNixon** [comment]
> Why do I have to confirm that I want to write to my MP and councillor.

> I sent the email then you ask me if I want to confirm. That is stupid.

> When I send I want it sent

**Comment by MyfanwyNixon** [comment]
>I got this email from frank turner requesting that i write to my mp this morning, attaching your site - so i took the 20 minutes out of my busy day to do so.....
Ive just got home to find your email that seems to indicate that for some reason it looks like my letter was not sent (you want me to reconfirm that i do want the letter sent)

> This step was not clear at all on your site, so i thought that id completed all that was needed. Now i feel like i completely wasted my time on this.

>Why didn't i just go to my mps page and just email him from there - seems like it would be simpler?

**Comment by MyfanwyNixon** [comment]
> message asking me if i really wanted to write to my MP after
     all....well call me Mr indecisive, but if I take the trouble to
     write to someone it's because that's what I want to do - i.e.
     WRITE TO THEM
 > Please do not patronise the electorate by asking them if they
     are serious...is this some kind of Kindergarten scheme?

**Comment by dracos** [comment]
"(From a doctor, so presumably someone with a reasonable education!)" <-- ha ha, no.
Sounds like the copy on either the please check your email or the confirmation email itself should be tweaked in some way?

**Comment by MyfanwyNixon** [comment]
Yes, I think we should look at both, and probably mostly the tone of the confirmation email. Maybe some experiments are in order.

I do wonder if these are people who are (somehow?) unfamiliar with the concept of confirmation, although they are a small percentage of users.

**Comment by davewhiteland** [comment]
My suspicion is that the confusion about confirmation is because, increasingly, people are used to sending email from a web-based client, so the idea that sending an email from within a browser needs confirmation is itself perhaps becoming *more* unexpected. (It's the web != internet understanding that almost nobody non-technical gets).  Conversely, @dracos may remember, as I do, the difficulty we had convincing an angry councillor in [a place in the UK] that although, yes, he *could* put anyone's email address in the box, *because of the confirmation email* he wasn't really going to be able to send his message [petition, in that case] from someone else's email address. Which of course is the whole point of the confirmation.

But it's not obvious. Unhelpfully, if we were to ever provide auth via another route (e.g., login with your Gmail account, etc) this problem doesn't go away, because then some people will continue to be bemused/enraged that the message is then going through an intermediary. So it's... tricky to get this right for the minority of people who think *they* are emailing their representative, rather than WTT doing so. 

But this does all suggest that, somewhere, we do need to be more explicit about the excellent reasons  WTT cannot simply be a list of `mailto:` links.  A "why?" link in the confirmation email might be a good idea, except for the risk of that link being misunderstood/mis-expected for the confirmation link; so maybe it really can only be explanatory text in the message body.

**Comment by dracos** [comment]
I doubt the numbers are large enough for any testing to show meaningful differences; as you say these are a small percentage in comparison. I think just making any improvements warranted and seeing what results; I also think there will always be some few who will need help regardless of what we do/say. There have been tickets open to look at/change these sorts of texts for some time, e.g. #68 (confirmation email) #141 (all emails) #164 ("100%" ref in failed email).

From a brief look at the check your email page, my suggestion would be to add a short reason to the paragraph explaining why they need to check their email; that it's to prevent someone pretending to be them. That seems to be all that's missing, it currently just reads as "do this, because we say so", which isn't great, and I think would explain most of the quoted things above if you're of a certain mind! https://www.writetothem.com/about-qa#whyconf has lots of explanation, it could link to that as part of it. The email itself could also explain why it exists, perhaps around the 'If you did not request' section, perhaps somewhere else with a more general rewrite.

**Comment by davewhiteland** [comment]
"it's to prevent someone pretending to be them" :+1: -- yes, that is the nub of it

**Comment by MyfanwyNixon** [comment]
> a short reason to the paragraph explaining why they need to check their email; that it's to prevent someone pretending to be them. 

Can't hurt, although it does assume people read the page at all. I'm assuming Mr "I've just got home to find your email" didn't!

**Comment by MyfanwyNixon** [comment]
But anyway yes, I will assign myself the task of writing some new copy.

**Comment by MyfanwyNixon** [comment]
I suggest this change to the 'now check your email' page:

**Now check your email!**
We've sent you a confirmation link. Click this and your message will be sent.

**Why?** 
The confirmation link ensures that no-one else is using your email address to send messages to your [rep]. 

It also acts as a check that you have entered the correct email address, so that any responses from your [rep] will come to the right place.

**Can't find the confirmation mail?**
It may take a few minutes to arrive. If it still hasn't come, please check your 'junk' or 'spam' folder.

**Changed your mind?**
If you don't click the link, your message will remain unsent. We'll send you one further reminder, and if you ignore that too, in time your message will be wiped from our servers.

**Comment by davewhiteland** [comment]
I don't think "Changed your mind?" is quite right; it's starting on the assumption they get what is going on here and are choosing not to click. Maybe that's way down the line in the thought process: speculation follows:

Perhaps the fundamental message here needs to be "Your message has **not been sent**. We will not send it **unless you click the link** we've sent you." 

I suggest we let users work out for themselves why they might or might not decide to click it (ie "Changed your mind?"); but before that makes sense, before there's any discussion as to what to do, the primary message that is lacking currently is: your message has not been sent. 

Which is to say, to address the people who are being tripped up by the issue is addressing, I believe the problem isn't really that they don't know they have to click the link, but that they think their message has already been sent (so, similarly, I don't think we should say "will remain unsent" either; that's remaining in a state they don't realise is it in).

So although "Now check your mail" is good and direct, perhaps that should be "We have not sent your message: check your mail"?

This feels a lot like something that would benefit from A/B testing, no?

**Comment by dracos** [comment]
I'm not sure, as it's such a small proportion already it would be hard to spot anything beyond variance. Also, for the issue I mention above (saying why), that is clearly something missing that should be added. I agree changed your mind text is probably better put in the email itself (where I think it does mention this already).

I would add that one possible source of confusion is the breadcrumbs, that don't have a "check your email" step and say "send message" when you're on the check your email page...

Anyway, in terms of text, my preference would be something like:

> Nearly done! Check your email to send your message.
> 
> We've sent you a confirmation link. Click this and your message will be sent.
> 
> Why?
> The confirmation link prevents anyone else from using your email address to send messages to your [rep]. It also acts as a check that you have entered the correct email address, so that any responses from your [rep] will come to the right place.
> 
> Can't find the confirmation mail?
> It may take a few minutes to arrive, please be patient. If it still hasn't come, please check your 'junk' or 'spam' folder.

**Comment by davewhiteland** [comment]
ooh yes acknowledging it as a step in the breadcrumbs is a good point :+1:

**Comment by MyfanwyNixon** [comment]
Yes, I agree about the breadcrumbs, that'd help.
I also feel like the only A/B testing you could do would be whether it cuts down on the cross emails, and they are unpredictable as well as being in low numbers.

I'm happy to drop the concept of changing your mind/not wanting to send - it occurs in the confirmation message anyway. And I fear that a 'Your message has not been sent' would look like an error to the skim-reader. So in short, :+1: to Matthew's suggested text.

**Comment by davewhiteland** [comment]
:+1:

**Comment by MyfanwyNixon** [comment]
Was going to (finally) get around to making a pull request to reflect these decisions but I'm not confident that I wouldn't break the code on this page.
